addons/l10n_ua/data/template/account.tax-ua_psbo.csv

```diff
@@ -1,79 +1,79 @@
+"","","","","","","","","","tax","refund","ua_psbp_6432"
+"sale_tax_template_vat14incl_psbo","10","в т. ч. ПДВ 14%","","в т. ч. ПДВ 14%","14.0","sale","tax_group_vat14","True","base","invoice",""
+"","","","","","","","","","tax","invoice","ua_psbp_6432"
 "","","","","","","","","","base","refund",""
-"","","","","","","","","","tax","refund","ua_psbp_6431"
-"sale_tax_template_vat7_psbo","11","Реалізація ПДВ 7%","","+ ПДВ 7%","7.0","sale","tax_group_vat7","","base","invoice",""
-"","","","","","","","","","tax","invoice","ua_psbp_6431"
+"","","","","","","","","","tax","refund","ua_psbp_6432"
+"sale_tax_template_vat7_psbo","11","ПДВ 7%","","+ ПДВ 7%","7.0","sale","tax_group_vat7","","base","invoice",""
+"","","","","","","","","","tax","invoice","ua_psbp_6432"
 "","","","","","","","","","base","refund",""
-"","","","","","","","","","tax","refund","ua_psbp_6431"
-"sale_tax_template_vat7incl_psbo","11","Реалізація в т. ч. ПДВ 7%","","в т .ч. ПДВ 7%","7.0","sale","tax_group_vat7","True","base","invoice",""
-"","","","","","","","","","tax","invoice","ua_psbp_6431"
+"","","","","","","","","","tax","refund","ua_psbp_6432"
+"sale_tax_template_vat7incl_psbo","11","в т. ч. ПДВ 7%","","в т .ч. ПДВ 7%","7.0","sale","tax_group_vat7","True","base","invoice",""
+"","","","","","","","","","tax","invoice","ua_psbp_6432"
 "","","","","","","","","","base","refund",""
-"","","","","","","","","","tax","refund","ua_psbp_6431"
-"sale_tax_template_vat0_psbo","12","Реалізація ПДВ 0%","","ПДВ 0%","0.0","sale","","","base","invoice",""
+"","","","","","","","","","tax","refund","ua_psbp_6432"
+"sale_tax_template_vat0_psbo","12","ПДВ 0%","","ПДВ 0%","0.0","sale","","","base","invoice",""
 "","","","","","","","","","tax","invoice",""
 "","","","","","","","","","base","refund",""
 "","","","","","","","","","tax","refund",""
-"sale_tax_template_vat_free_psbo","13","Реалізація звільнена від  ПДВ","","Звільнено від ПДВ","0.0","sale","","","base","invoice",""
+"sale_tax_template_vat_free_psbo","13","Звільнено від  ПДВ","","Звільнено від ПДВ","0.0","sale","","","base","invoice",""
 "","","","","","","","","","tax","invoice",""
 "","","","","","","","","","base","refund",""
 "","","","","","","","","","tax","refund",""
-"sale_tax_template_vat_not_psbo","14","Реалізація Не є об'єктом ПДВ","","Не є об'єктом ПДВ","0.0","sale","","","base","invoice",""
+"sale_tax_template_vat_not_psbo","14","Не є об'єктом ПДВ","","Не є об'єктом ПДВ","0.0","sale","","","base","invoice",""
 "","","","","","","","","","tax","invoice",""
 "","","","","","","","","","base","refund",""
 "","","","","","","","","","tax","refund",""
-"purchase_tax_template_vat20_psbo","19","Придбання ПДВ 20%","","+ ПДВ 20%","20.0","purchase","tax_group_vat20","","base","invoice",""
-"","","","","","","","","","tax","invoice","ua_psbp_6441"
+"purchase_tax_template_vat20_psbo","19","ПДВ 20%","","+ ПДВ 20%","20.0","purchase","tax_group_vat20","","base","invoice",""

```

Comment:
Does this mean, the 6442 account should rather be used on the tax group as tax receivable account? For the tax closing entry to work out the box in Ukraine?